File: SGEM/Models/Persona.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SGEM.Models
{
    public class Persona: Base
    {
        public Cargo Cargo { get; set; }
        public string Nombres { get; set; }
        public string Apellidos { get; set; }
        public string TipoDocumento { get; set; }
        public string NumeroDocumento { get; set; }
        public string Distrito { get; set; }
        public string Direccion { get; set; }
    }
}

File: SGEM/ClientApp/src/app/personas/persona.ts
import { ICargo } from "../cargos/cargo";

export interface IPersona {
  id?: number;
  nombres?: string;
  apellidos?: string;
  cargo?: ICargo;
  tipoDocumento?: string;
  numeroDocumento?: string;
  distrito?: string;
  direccion?: string;
}
